voxblox/include/voxblox/io/sdf_ply.h

```diff
@@ -31,18 +31,18 @@ enum PlyOutputTypes {
 // SDF values within this max value result in a colored point.
 template <typename VoxelType>
 bool getColorFromVoxel(const VoxelType& voxel,
-                       const FloatingPoint sdf_color_range,
-                       const FloatingPoint sdf_max_value, Color* color);
+                       const float sdf_color_range,
```

Comment:
Ahhh right, since the SDF distances are always floats anyway. This makes more sense.